### PR TITLE
added test for convienece functions. Also fixed Instance indentation

### PR DESCRIPTION
**Pull request by cloudformdesign**

### CODE REVIEW

File: pynio/instance.py [hunk 1]
```diff
@@ -51,3 +56,39 @@ def _get_services(self):
                                   config=resp[s],
                                   instance=self)
         return services
+
+    def create_block(self, name, type, config=None):
+        '''Convenience function to create a block and add it to instance'''
+        return Block(name, type, config, instance=self)
```
Comment (hansmosh) [inline pynio/instance.py]: Shouldn't it actually add the block to the instance too?

File: pynio/instance.py [hunk 2]
```diff
@@ -51,3 +56,39 @@ def _get_services(self):
...
+
+    def create_service(self, name, type=None, config=None):
+        '''Convenience function to create a service and add it to instance'''
+        if type is None:
+            return Service(name, config=config, instance=self)
+        else:
+            return Service(name, type, config=config, instance=self)
```
Comment (hansmosh) [inline pynio/instance.py]: Same as above

File: pynio/instance.py (hunk 1 above)
Comment (cloudformdesign) [inline pynio/instance.py]: isn't that what `instance=self` does?

File: pynio/instance.py (hunk 2 above)
Comment (cloudformdesign) [inline pynio/instance.py]: same as above :P

File: pynio/instance.py (hunk 1 above)
Comment (cloudformdesign) [inline pynio/instance.py]: actually, crap! I did it right before by passing instance=self._instance!

Comment (cloudformdesign) [inline pynio/instance.py]: whoops, you were right for now.

**Comment by cloudformdesign** [comment]
this should be ready now

**Pull request merged**